schemas/json/layout/layout.schema.v1.json

```
@@ -160,13 +160,13 @@
         "minDate": {
           "type": "string",
           "title": "Minimum allowed date",
-          "description": "Sets the minimum allowed date. Defaults to 1900-01-01T12:00:00.000Z'.",
+          "description": "Sets the minimum allowed date. Can also use keywords such as 'today'. The 'today' keyword will disable all past dates dynamically. Defaults to 1900-01-01T12:00:00.000Z'.",
```

`Can also use keywords such as 'today'. The 'today' keyword will disable all past dates dynamically.`
This makes is sound like we support other keywords. I suggest we only reference the once we actually support, and update this if the need be in the future.

Better with something like this?

```suggestion
          "description": "Sets the minimum allowed date. Can also use keyword 'today' to disable all past dates dynamically based on the current date. Defaults to 1900-01-01T12:00:00.000Z'.",
```